repo/meta/index.json

```diff
@@ -31,6 +31,18 @@
       "versions":{
         "alpha":"0"
       }
+    },
+    {
+      "currentVersion":"1.3.0",
+      "description":"Spark is a fast and general cluster computing system for Big Data",
+      "name":"spark",
+      "tags":[
+        "mesosphere",
+        "framework"
```

Comment:
Might be nice to have a bigdata/machinelearning tag or something like that. Not sure if that's going against the intended usage of tags though (what do you think, @ConnorDoyle?).